.github/workflows/create-tag-from-mix-version.yaml

```diff
@@ -0,0 +1,36 @@
+name: create-tag-from-mix-version
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        required: false
+        type: string
+        default: ubuntu-latest
+jobs:
+  create-tag-from-mix-file:
+    name: Create Tag from Mix Version
+    if: github.event.pull_request.merged == true
+    runs-on: ${{ inputs.runs-on }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up git
+        id: git_setup
+        run: |
+          git config --global user.email "rentpath-workflow-bot.idg.dx@rentpath.com"
+          git config --global user.name "rentpath-workflow-bot"
+      - name: Get the version
```

Comment:
Same comment about `name`.  Also, we use hyphens instead of underscores in `id` and `output` identifiers, so `get-version` and `mix-version` are preferred. Actually, since this step is setting the `mix-version` output, it would be consistent with https://github.com/rentpath/actions-services-workflows/blob/main/.github/workflows/deployment-images.yaml#L62 to `id` this step as `set-mix-version`.